src/index.js

```diff
@@ -1,7 +1,10 @@
+// @flow
+
 import Camera from './Camera';
-import RNCamera from './RNCamera';
+import RNCamera, { type Status as CameraStatus } from './RNCamera';
 import FaceDetector from './FaceDetector';
 
+export type CameraStatus;
```

Comment:
```suggestion
export type CameraStatus = CameraStatus;
```

You will have to change this